plugins/janus_recordplay.c

```diff
@@ -448,6 +450,38 @@ typedef struct janus_recordplay_session {
 static GHashTable *sessions;
 static janus_mutex sessions_mutex = JANUS_MUTEX_INITIALIZER;
 
+static GList *janus_recordplay_filter_extmaps(const GList *attributes) {
+	GList *filtered = NULL;
+	while(attributes != NULL) {
+		janus_sdp_attribute *attr = (janus_sdp_attribute *)attributes->data;
+		if(!strcasecmp(attr->name, "extmap"))
```

Comment:
Action is multiline, per code style there should be brackets (less ambiguous, more readable).